packages/blade/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,57 @@
+import type { ReactElement } from 'react';
+import { cloneElement, Children } from 'react';
+import { BaseBox } from '~components/Box/BaseBox';
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import type { TestID } from '~src/_helpers/types';
+
+type AccordionProps = {
+  /**
+   * Makes the passed item index expanded by default (uncontrolled)
+   */
+  defaultExpandedIndex?: number;
+
+  /**
+   * Expands the passed index (controlled)
+   */
+  expandedIndex?: number;
+
+  /**
+   * Callback for change in any item's expanded state
+   */
+  onChange?: ({ expandedIndex }: { expandedIndex: number | undefined }) => void;
+
+  /**
+   * Adds numeric index at the beginning of items
+   *
+   * @default false
+   */
+  showNumberPrefix?: boolean;
+
+  /**
+   * Accepts `AccordionItem` child nodes
+   */
+  children: ReactElement | ReactElement[];
```

Comment:
```suggestion
  children: React.ReactNode
  ```

Reply:
ReactNode is bit relaxed, I intentionally want it to be able to only accept JSX children. Any other suggestions you think would fit better?

Reply:
Oh I see it should only contain AccordionItem components is it? Are we doing runtime checks for this?

Reply:
Haven't added any runtime checks for that, I was thinking it should be fine because the UI will break clearly in case of misuse.